packages/grafana-runtime/src/services/live.ts

```diff
@@ -48,6 +48,8 @@ export interface GrafanaLiveSrv {
    * Send data to a channel.  This feature is disabled for most channels and will return an error
    */
   publish<T>(channel: string, data: any): Promise<T>;
+
+  addListener(event: string | symbol, listener: (...args: any[]) => void): void;
```

Comment:
what are possible event values?  for our API, should they be more specific than string|symbol?

Reply:
This is a good question, I just copied the call signature of the corresponding centrifuge method but you're right that there could be more specificity. I'll look into it

Reply:
It seems like arbitrarily named events can be emitted with `centrifuge.emit`, but I suppose from the perspective of a consumer of the grafana live API that flexibility might not be needed or advisable. We could limit it just to the events that centrifuge-js itself makes use of (publish, join, leave, subscribe, unsubscribe, error, message, connect, disconnect)?

Reply:
Not sure what the purpose of this listener is we have the getChannelStream for this exact purpose no?